dbms/src/Common/HashTable/Hash.h

```diff
@@ -39,7 +39,7 @@ inline DB::UInt64 intHash64(DB::UInt64 x)
 #include <nmmintrin.h>
 #endif
 
-#if __aarch64__
+#if __aarch64__ && __ARM_FEATURE_CRC32
```

Comment:
Doesn't AArch64 implies the existence of CRC32 instructions?
Are you sure that this macro is available when we compile for AArch64?